python/feldera/rest/config.py

```diff
@@ -6,7 +6,8 @@
 
 class Config:
     """
-    :class:`.FelderaClient`'s credentials and configuration parameters
+    :class:`.FelderaClient` configuration, which encompasses where to reach the Feldera API,

```

Comment:
which includes authentication information and the address of the Feldera API the client will interact with